coldfront/plugins/qumulo/management/commands/migrate_project_attributes.py

```diff
@@ -0,0 +1,38 @@
+from coldfront.core.project.models import (
+    ProjectAttributeType,
+    AttributeType,
+    Project,
+    ProjectAttribute,
+)
+from django.core.management.base import BaseCommand
+
+from django.db.models import OuterRef, Subquery
+
+
+class Command(BaseCommand):
+    def handle(self, *args, **options):
+        print("Updating Qumulo Project Attributes")
+        # updating required project attributes for all projects
+
+        # for sponsor department number use "unknown"
+        # can be updated manually later if necessary
+        self._migrate_project_attribute("sponsor_department_number", "unknown")
```

Comment:
Since these are supposed to be default values, we need to add them to the attribute creation of new allocations.